tests/functional/strategies/test_proxy_exploit.py

```diff
@@ -0,0 +1,63 @@
+import time
```

Comment:
This file can be removed once the fix is verified?

Reply:
Yes, removing the test makes more sense.